tests/frontend-storage/src/backend/server-storage-proxy/Server.ts

```diff
@@ -0,0 +1,138 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as path from "path";
+
+import * as express from "express";
+import { Container } from "inversify";
+
+import { Bindable } from "@itwin/cloud-agnostic-core";
+import {
+  ServerStorage,
+  ServerStorageDependency,
+} from "@itwin/object-storage-core";
+
+import * as Common from "./Common";
+
+export class ServerStorageProxyBackend extends Bindable {
```

Comment:
I already saw there was a comment about filenames not matching class names and I think this is applicable here too - there's only one exported symbol here and for consistency the file should be named after the class.